consensus/src/genesis/config.rs

```diff
@@ -0,0 +1,542 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0
+
+use mina_crypto::{
+    base58::{version_bytes, Base58Encodable, Base58EncodableHash},
+    hash::*,
+    signature::PublicKey,
+};
+use mina_rs_base::{
+    finite_ec_point, finite_ec_point_pair,
+    types::{proof_messages::ProofMessageWithoutDegreeBoundList, *},
+};
+
+const ERR_FAIL_TO_DECODE_B58: &str = "Failed to decode hash from base58";
```

Comment:
This doesn't feel idiomatic to me. I guess it's ok since this for the given values we can be sure this function can't error. Maybe add some comments justifying why this is how you have done it this way so a new dev doesn't copy this error handling pattern somewhere else :P

Reply:
Justification added